src/module.c

```diff
@@ -6842,6 +6843,20 @@ int RM_ACLCheckChannelPermissions(RedisModuleUserID *userid, RedisModuleString *
     return REDISMODULE_OK;
 }
 
+/* Adds a new entry in the ACL log.
+ * Returns REDISMODULE_OK on success and REDISMODULE_ERR on error. */
+int RM_ACLAddLogEntry(RedisModuleCtx *ctx, RedisModuleUserID *userid, RedisModuleString *object) {
```

Comment:
we need to document what's `object`.
i see the ACL LOG doesn't really define that
https://redis.io/commands/acl-log
so maybe it's just sufficient to give a reference to there?

i see that there's also a `context` field that can be set to `toplevel`, `multi`, or `lua`, i suppose that in this case we either use a hard coded `module`, use the module name, or expose that as an argument in this function?
whatever we decide, this also applies for the logging we added in RM_Call.

i vote for a hard coded `module` @yossigo ?

Reply:
i see this API isn't covered by the test.

Reply:
Currently, we are taking the context from the ctx->client, so if a client is running some module command inside multi or lua script (if it's valid) you will see these contexts. if it doesn't gain anything I agree that we can use the 'module name' or just "module" context.

The API does tested, pls see in "test module check acl for command perm" (rm_call_aclcheck_cmd)

Reply:
i think seeing a module context is more valuable than seeing multi or Lua, but maybe we wanna change the context thing to flags rather than enum and show both.